### ISSUE TITLE
unsquashfs: error while loading library 'liblzma.so.5'

### ISSUE THREAD

**Issue by jens-maus** [Thinstation/thinstation#70]
when building PXE images with the current 'master' branch of thinstation unsquashing of the lib.squash file fails because initrd seems to be missing the required liblzma.so.5 library file, thus the "unsquashfs" command throws an error accordingly.

**Comment by Thinstation** [comment]
Are you at the latest checkout? I just checked, and liblzma is there.

**Comment by jens-maus** [comment]
I am sorry, but this issue doesn't seem to be resolved. Of course I am working with the latest HEAD of your master branch (after having merge it into our own hzdr/thinstation fork) and I still get an initrd file where liblzma.so.5 is missing. See here:

```
maus@titan:/tftpboot/thinstation$ ./lsinitrd initrd-1.8 | grep lzma
bin/lzma
bin/unlzma
```

Performing the same with a checkout several days earlier I can perfectly see liblibzma.so being part of the initrd file:

```
maus@titan:/tftpboot/thinstation$ ./lsinitrd initrd-1.7 | grep lzma
bin/lzma
bin/unlzma
lib/liblzma.so.5
```

Thus, it seems something is preventing liblzma.so from being added to the initrd file properly?!?

In addition I just spotted that after having updated our master with your changes from yesterday the perl locale warning comes up again during the build:

```
Adding Library Dependencies

perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
	LANGUAGE = "POSIX",
	LC_ALL = (unset),
	LC_COLLATE = "POSIX",
	LC_CTYPE = "en_US.UTF-8",
	LC_TIME = "C",
	LANG = "C"
    are supported and installed on your system.
perl: warning: Falling back to the standard locale ("C").
[...]
```

And this continues until all library dependencies have been added...

**Comment by jens-maus** [comment]
Ok, I think I have found the root of liblzma.so.5 being missing. It seems there were a workaround for this which you removed (commented out recently).

See here:
https://github.com/Thinstation/thinstation/blob/master/ts/build/fastboot/fastboot-mangle#L127

I just uncommented that line here and now liblzma.so.5 is part of initrd again. Thought something else might be wrong since it isn't directly added in the beginning of fastboot-mangle even that it is listed in the 'lib-boot' file.
